Actions.xcodeproj/project.pbxproj

```diff
@@ -1074,7 +1089,7 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = YG56YK5RN5;
+				DEVELOPMENT_TEAM = ME2UKFKL5H;
```

Comment:
Don't commit this.

Reply:
I've added a config property for this, so it's easier not to commit this